test/core/templaters/placeholder_test.py

```diff
@@ -306,6 +322,8 @@ def test__templater_raw():
         "numeric_dollar_with_braces_and_string",
         "percent",
         "ampersand",
+        "flyway_var",
+        "flyway_var",
```

Comment:
Why does `flyway_var` appear twice?

Reply:
I saw some warning to the effect that `ids` list has less elements than the test sql's list - let me remove and retest